perl/default.nix

```
@@ -11,24 +21,21 @@ perl.pkgs.toPerlModule (stdenv.mkDerivation (finalAttrs: {
   src = fileset.toSource {
     root = ../.;
```

This can become
```suggestion
    root = ./.;
```
now